README.md

```diff
@@ -7,7 +7,17 @@ Visit the website: [up-for-grabs.net](http://up-for-grabs.net/)
 
 ## Testing the site locally
 
-Most of the changes should require updates to the [projects.yml](https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects.yml) file.
+Each of the projects exists under the [projects](https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects/) folder - simply add a new file (the name isn't displayed in the UI, so don't stress about it) and fill out these values:
```

Comment:
> the name isn't displayed in the UI, so don't stress about it

We may want to reserve the right to use the filename in the future, e.g. to support deep links like `http://up-for-grabs.net/#/projects/up-for-grabs`.